File: Plugins/Aspose_Diagram_Java_for_Ruby/lib/asposediagramjava/Diagrams/creatediagram.rb
module Asposediagramjava
  module CreateDiagram
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'

        # Create instance of Diagram
        diagram = Rjb::import('com.aspose.diagram.Diagram').new

        # Save as VDX
        diagram.save(data_dir + "CreateDiagram.vdx", Rjb::import('com.aspose.diagram.SaveFileFormat').VDX)

        puts "Created visio diagram successfully!"
    end
  end
end

File: Plugins/Aspose_Diagram_Java_for_PHP/src/aspose/diagram/WorkingwithDiagrams/CreateDiagram.php
<?php
namespace Aspose\Diagram\WorkingwithDiagrams;

use com\aspose\diagram\Diagram as Diagram;
use com\aspose\diagram\SaveFileFormat as SaveFileFormat;
class CreateDiagram{

    public static function run($dataDir=null){
        # Create instance of Diagram
        $diagram = new Diagram();

        # Save as VDX
        $saveFileFormat=new SaveFileFormat();
        $diagram->save($dataDir."CreateDiagram.vdx", $saveFileFormat->VDX);
        print "Created visio diagram successfully!".PHP_EOL;
    }
}